src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -42,6 +42,7 @@ class OutputFormat(Enum):
     OPENAI_CHAT_COMPLETIONS = auto()
     OPENAI_COMPLETIONS = auto()
     OPENAI_EMBEDDINGS = auto()
+    OPENAI_RANKINGS = auto()
```

Comment:
I think this is going to be misleading, as OpenAI has no ranking API. With the current breakdown (OpenAI vs Triton), it goes best under the endpoint-type though. I'll think about this a bit. We might be able to get away with this and just leave a comment if there is not a better option without a larger refactor.

Reply:
Could you use `RANKINGS` for now? And for service-kind, can you add an option and use `none` (or `misc` if we think that's better)?

CC: @nv-hwoo to weigh in and because this enum is duplicated in metrics until we refactor.